charts/cyclecloud/templates/Deployment.yaml

```diff
@@ -78,7 +80,7 @@ spec:
         command: ["/bin/bash"]
         securityContext:
           allowPrivilegeEscalation: false  
-        args: ["-c", "mkdir -p /mnt/cycle_server/data /mnt/cycle_server/logs /mnt/cycle_server/.ssh && sed -i \"s/webServerMaxHeapSize\\=.*/webServerMaxHeapSize\\=$JAVA_HEAP_SIZE/\" /opt/cycle_server/config/cycle_server.properties && chmod 700 /mnt/cycle_server/.ssh && chmod +x scripts/run_cyclecloud.sh && ./scripts/run_cyclecloud.sh"]
+        args: ["-c", "mkdir -p /mnt/cycle_server/data /mnt/cycle_server/logs /mnt/cycle_server/.ssh && sed -i \"s/webServerMaxHeapSize\\=.*/webServerMaxHeapSize\\=$CYCLECLOUD_WEBSERVER_MAX_HEAP_SIZE/\" /opt/cycle_server/config/cycle_server.properties && chmod 700 /mnt/cycle_server/.ssh && chmod +x scripts/run_cyclecloud.sh && ./scripts/run_cyclecloud.sh"]
```

Comment:
I don't know why we want the sed command now - we are always updating the heap size in cyclecloud_install.py, no?

Reply:
I think we can get rid of sed command since now the run_cyclecloud.py would correctly update the HEAPSIZE

Reply:
Good catch